app/styles/main.scss

```diff
@@ -392,6 +391,10 @@ div.cartodb-share a{
   -o-transition: all 250ms ease-in;
   transition: all 250ms ease-in;
 }
+.link {
+  color: #337ab7;
```

Comment:
please add this colour to `_variables.scss` or use one of the already-defined blues there